packages/hub/config/knexfile.js

```diff
@@ -0,0 +1,38 @@
+const path = require('path');
```

Comment:
Why the switch to JavaScript over TypeScript?

Reply:
Good question. I tried moving `knexfile.ts` to the `config` directory from the root package directory and saw this error `File 'statechannels/monorepo/packages/hub/config/knexfile.ts' is not under 'rootDir' 'statechannels/monorepo/packages/hub/src'. 'rootDir' is expected to contain all source files.`

`knexfile.ts` probably doesn't belong in the `src` directory. One option is to change the tsconfig settings. I adopted the path of least resistance and just made the file a `js` file since it doesn't use any typescript constructs anyways.

Reply:
Does it need to be in the root directory? Seems like there is a `typeof import("@types/knex").Config` type we could typecast the config as.

Reply:
knexfile does not need to be in the root package directory, nor is it in the pull request. But, Typescript compiler is unhappy with any typescript files outside of the `tsconfig` rootDir, which is set to `src`.

Good call on knex types. I will look into those.